tools/crm_mon_output.c

```diff
@@ -827,6 +913,102 @@ node_attribute_xml(pcmk__output_t *out, va_list args) {
     return 0;
 }
 
+static int
+op_history_html(pcmk__output_t *out, va_list args) {
+    xmlNode *xml_op = va_arg(args, xmlNode *);
+    const char *task = va_arg(args, const char *);
+    const char *interval_ms_s = va_arg(args, const char *);
+    int rc = va_arg(args, int);
+    unsigned int mon_ops = va_arg(args, unsigned int);
+
+    char *buf = op_history_string(xml_op, task, interval_ms_s, rc, mon_ops);
+
+    pcmk__output_create_html_node(out, "li", NULL, NULL, buf);
+
+    free(buf);
+    return 0;
+}
+
+static int
+op_history_text(pcmk__output_t *out, va_list args) {
+    xmlNode *xml_op = va_arg(args, xmlNode *);
+    const char *task = va_arg(args, const char *);
+    const char *interval_ms_s = va_arg(args, const char *);
+    int rc = va_arg(args, int);
+    unsigned int mon_ops = va_arg(args, unsigned int);
+
+    char *buf = op_history_string(xml_op, task, interval_ms_s, rc, mon_ops);
+
+    out->list_item(out, NULL, buf);
+
+    free(buf);
+    return 0;
+}
```

Comment:
Can't this function be used for both text and html?

Reply:
Yep.